src/main/java/org/sunbird/cassandra/utils/CassandraOperationImpl.java

```
@@ -155,10 +157,8 @@ public void deleteRecord(String keyspaceName, String tableName, Map<String, Obje
             });
             connectionManager.getSession(keyspaceName).execute(delete);
         } catch (Exception e) {
-            logger.error("CassandraOperationImpl: deleteRecord by composite key. " + Constants.EXCEPTION_MSG_DELETE
-                    + tableName + " : " + e.getMessage(), e);
-            throw e;
-
+        	logger.error(String.format("CassandraOperationImpl: deleteRecord by composite key. %s %s %s",
+					Constants.EXCEPTION_MSG_DELETE, tableName, e.getMessage()));
```

Need to throw the exception - so that caller knows the issues...